kalite/__init__.py

```diff
@@ -27,3 +28,8 @@ def is_installed():
         # except:
         #     return False
         return True
+
+
+# Modify Request's default user agent to include the KA Lite version number
+base_headers = requests.defaults.defaults["base_headers"]
+base_headers["User-Agent"] = ("ka-lite/%s " % VERSION) + base_headers["User-Agent"]
```

Comment:
Turns out the requests code I had been inspecting was for a different version than we use in KA Lite. The version we use allows the above approach.